packages/dds/tree/src/tableSchema.ts

```diff
@@ -339,33 +389,46 @@ export namespace System_TableSchema {
 
 	// #region Table
 
+	/**
+	 * Base options for creating table schema.
+	 * @remarks Includes parameters common to all table factory overloads.
+	 * @system @internal
+	 */
+	export type TableFactoryOptionsBase<
+		TScope extends string | undefined = string | undefined,
+		TCell extends ImplicitAllowedTypes = ImplicitAllowedTypes,
+	> = OptionsWithSchemaFactory<TScope> & OptionsWithCellSchema<TCell>;
+
 	/**
 	 * Factory for creating new table schema.
 	 * @system @internal
 	 */
 	// eslint-disable-next-line @typescript-eslint/explicit-function-return-type -- Return type is too complex to be reasonable to specify
 	export function createTableInternal<
 		const TInputScope extends string | undefined,
-		const TCell extends ImplicitAllowedTypes,
-		const TColumn extends ColumnSchemaBase<TInputScope> = ColumnSchemaBase<TInputScope>,
-		const TRow extends RowSchemaBase<TInputScope, TCell> = RowSchemaBase<TInputScope, TCell>,
+		const TCellSchema extends ImplicitAllowedTypes,
```

Comment:
Explanation of type parameter name changes: I simplified the type param names for the public-facing APIs to omit the "Schema" postfix. But for the internal functions, where we derive value and insertable types, I ensured the type parameters _do_ include the postfix, as it helps differentiate the various types involved at a glance.